include/assimp/mesh.h

```diff
@@ -268,6 +271,16 @@ struct aiBone {
     //! The maximum value for this member is #AI_MAX_BONE_WEIGHTS.
     unsigned int mNumWeights;
 
+#ifndef ASSIMP_BUILD_NO_ARMATUREPOPULATE_PROCESS
+    // The bone armature node - used for skeleton conversion
+    // you must enable aiProcess_PopulateArmatureData to populate this
+    C_STRUCT aiNode* mArmature;
+
+    // The bone node in the scene - used for skeleton conversion
+    // you must enable aiProcess_PopulateArmatureData to populate this
+    C_STRUCT aiNode* mNode;
+
+#endif
     //! The influence weights of this bone, by vertex index.
     C_STRUCT aiVertexWeight* mWeights;
 
```

Comment:
Compile-time ABI changes without also changing version or something in `aiGetCompileFlags` make shared libs harder to use safely. Could these pointers be included always, even if the extra processing isn't included?

Also, adding to the end of structures instead of the middle makes it slightly easier to support multiple versions of the library at once, since they can share a common prefix (or even whole thing if not included directly in another structure).